js/edit.js

```diff
@@ -13,35 +15,32 @@
     editColor: document.querySelector('#edit-color'),
     picker: document.querySelector('#edit-color gaia-color-picker'),
     iframe: document.querySelector('#edit iframe'),
+    autotheme: document.querySelector('#edit-color .autotheme-palette'),
 
     prepareForDisplay: function(params) {
       currentTheme = params.theme;
 
       this.title.textContent = params.group + ' / ' + params.section;
       this.header.setAttr('action', 'back');
+      AutoTheme.showPalette(this.autotheme);
 
-      this.picker.onchange = () => {
-        if (!currentKey) {
-          return;
-        }
-        var value = this.picker.value;
-        this.iframe.contentDocument.body.style.setProperty(currentKey, value);
-      };
+      this.picker.addEventListener('change', this.onPickerChange.bind(this));
 
       var currentList = this.list.querySelector('gaia-list');
       if (currentList) {
         this.list.removeChild(currentList);
       }
 
-      var list = document.createElement('gaia-list')
+      var list = document.createElement('gaia-list');
       currentSection = currentTheme.groups[params.group][params.section];
 
-      this.iframe.src = '/' + params.section + '-preview.html';
+      this.iframe.src = params.section + '-preview.html';
```

Comment:
IIRC this makes it work when loading the index file in Firefox without a server.